tests/test_context_processors.py

```diff
@@ -76,3 +76,58 @@ def test_header_footer_context_processor(settings):
         'header_footer_contact_us_url': 'http://bones.com',
         'header_footer_css_active_classes': {'fab': True},
     }
+
+
+def test_urls_processor(rf, settings):
+    settings.GREAT_HOME = 'http://home.com'
+    settings.GREAT_EXPORT_HOME = 'http://export.com'
+    settings.EXPORTING_NEW = 'http://export.com/new'
+    settings.EXPORTING_OCCASIONAL = 'http://export.com/occasional'
+    settings.EXPORTING_REGULAR = 'http://export.com/regular'
+    settings.GUIDANCE_MARKET_RESEARCH = 'http://market-research.com'
+    settings.GUIDANCE_CUSTOMER_INSIGHT = 'http://customer-insight.com'
+    settings.GUIDANCE_FINANCE = 'http://finance.com'
+    settings.GUIDANCE_BUSINESS_PLANNING = 'http://business-planning.com'
+    settings.GUIDANCE_GETTING_PAID = 'http://getting-paid.com'
+    settings.GUIDANCE_OPERATIONS_AND_COMPLIANCE = (
+        'http://operations-and-compliance.com')
+    settings.SERVICES_FAB = 'http://export.com/fab'
+    settings.SERVICES_SOO = 'http://export.com/soo'
+    settings.SERVICES_EXOPPS = 'http://export.com/exopps'
+    settings.SERVICES_GET_FINANCE = 'http://export.com/get-finance'
+    settings.SERVICES_EVENTS = 'http://export.com/events'
+    settings.INFO_ABOUT = 'http://about.com'
+    settings.INFO_CONTACT_US_DIRECTORY = 'http://contact.com'
+    settings.INFO_PRIVACY_AND_COOKIES = 'http://privacy-and-cookies.com'
+    settings.INFO_TERMS_AND_CONDITIONS = 'http://terms-and-conditions.com'
+    settings.INFO_DIT = 'http://dit.com'
+
+    actual = context_processors.urls_processor(None)
+
+    assert actual == {
```

Comment:
sorry I was not clear before: we need the following tests:

 1) confirm that the default values (as defined in the context processor) are used if values in `settings.py` are not defined.
 2) confirm that the values in settings.py are used if specified. 

To test the above most effectively, it would be prudent to remove the urls from `conftest.py`

Reply:
dang it. page hadn't updated with this so i merged already. will do this and make a new PR